openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -29,6 +30,7 @@ class AccountView(APIView):
 
         **Example Requests**
 
+            GET /api/user/v1/accounts?usernames={username1,username2}[?view=shared]
             GET /api/user/v1/accounts/{username}/[?view=shared]
```

Comment:
This actually looks like a promising solution: Can we make /api/user/v1/accounts/?usernames={username1,username2} ALWAYS return a list of dicts, even if only one username is requested, and /api/user/v1/accounts/{username}/ ALWAYS return a dict.  The API layer could always return a list, and the second form of the view can unwrap the user from the list.